tests/vmcheck/test-layering-scripts.sh

```diff
@@ -77,6 +77,38 @@ vm_cmd cat /usr/lib/prefixtest.txt > prefixtest.txt
 assert_file_has_content prefixtest.txt "/usr"
 echo "ok script expansion"
 
+vm_rpmostree rollback
+vm_reboot
+vm_rpmostree cleanup -p
+# We use /usr/share/licenses since it's small predictable content
+license_combos="zlib-rpm systemd-tar-rpm sed-tzdata"
+license_un_combos="zlib systemd-rpm"
+vm_build_rpm scriptpkg4 \
+             transfiletriggerin "/usr/share/licenses/zlib /usr/share/licenses/rpm" 'cat >/usr/share/transfiletriggerin-license-zlib-rpm.txt' \
```

Comment:
We can just `s/cat/sort` here right? (And the other ones) To avoid having to sort them down there.